spec/function.dd

```diff
@@ -1905,9 +1905,32 @@ $(H3 $(LNAME2 ref-return-scope-parameters, Ref Return Scope Parameters))
         When a parameter is passed by `ref` and has both the `return` and `scope` storage classes,
         it gets $(LINK2 #return-scope-parameters, `return scope`) semantics if and only if the `return` and `scope`
         keywords appear adjacent to each other, in that order.
-        Specifying a `return ref` and `scope` parameter enables returning a reference to a scope pointer.
-        In all other cases, the parameter has $(LINK2 #return-ref-parameters, `return ref`) semantics
-        and regular $(LINK2 #scope-parameters, `scope`) semantics.)
+        This behavior is summarized as follows:)
+
+        $(TABLE2 Combinations of `ref` $(COMMA) `return` and `scope`,
+        $(THEAD Order, Interpretation)
+        $(TROW `ref`,              $(REFP))
+        $(TROW `return`,           `return`)
```

Comment:
`return` is not an interpretation
```suggestion
        $(TROW `return`,           $(RETURN_SCOPE))
```

Reply:
It depends on the context. `See mtype.buildScopeRef`